examples/direct_fidelity_estimation_test.py

```diff
@@ -108,13 +110,17 @@ def test_same_pauli_traces_clifford():
         cirq.CNOT(qubits[0], qubits[2]),
     )
 
-    clifford_state = cirq.CliffordState(qubit_map={qubits[i]: i for i in range(len(qubits))})
+    qubit_map = {qubits[i]: i for i in range(n_qubits)}
+    clifford_tableau = cirq.CliffordTableau(n_qubits)
     for gate in circuit.all_operations():
-        clifford_state.apply_unitary(gate)
+        tableau_args = clifford.ActOnCliffordTableauArgs(
+            clifford_tableau, [qubit_map[i] for i in gate.qubits], np.random.RandomState(), {}
+        )
+        act_on(gate, tableau_args)
```

Comment:
The above lines in the test will always need to be kept in sync with the implementation exactly. This is usually a bad sign. Can we somehow test the equivalency of the two`_estimate_pauli_traces_` methods without duplicating the implementation here?

Maybe stick a no-op non-clifford gate at the start for the `general` one so that it doesn't pick up the clifford path? Not sure if such changes would affect this particular case. Let me know what you think.

Reply:
Yes, I can do that. However, I am not sure how to create a no-op gate that's not Clifford. Would you have some code pointers and then I'll do it.

Reply:
A `CCX` (Toffoli) gate at the beginning of the second circuit would be a no-op if the starting state is |000> (default). But I'm not very familiar with the direct fidelity estimation to know if it is sensitive to the initial state or tries to be agnostic to the initial state (if it's the latter it will not be a no-op).

Reply:
Done. Thanks for the tip.